hack/tools/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/cespare/prettybench v0.0.0-20150116022406-03b8cfe5406c
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.36.0
-	github.com/google/go-flow-levee v0.1.4-0.20201102181719-72c65d71b1d3
+	github.com/google/go-flow-levee v0.1.5-0.20210422150831-99e72693f5c8
```

Comment:
That SHA has since been tagged with `v0.1.5` release tag.  If you wanted to drop the date-and-SHA, you could.  There doesn't seemt o be any consistency here, though, since there are two other SHA-specified modules here.

Reply:
We prefer tags, because it makes it easier to track the delta, but not all dependencies have compatible go module versions.